i18n/en.json

```diff
@@ -196,10 +202,11 @@
 	"namespaces-protection": "Which userright should be needed to edit this namespace?",
 	"namespaces-search": "Should this namespace be searchable by default?",
 	"namespaces-subpages": "Should this namespace have subpages enabled?",
+	"permissions-advanced": "Advanced settings",
 	"permissions-assigned": "Assigned permissions",
 	"permissions-autopromote": "Autopromotion",
 	"permissions-delete": "Delete",
```

Comment:
We should be able to remove the section keys for both delete and autopromote now since it's being merged can't we?